api-specs/test/api.raml

```diff
@@ -271,6 +271,11 @@ uses:
             body:
               type: arrays.ArrayTestType
               example: !include examples/array-test-type.json
+          201:
```

Comment:
We could improve the "smoke-test-ish-ness" of this endpoint by using not only examples that are referred to "locally" at the endpoint but also omitting them in some (like our actual API spec mostly does) and having the kit render the generic example of that type.   

I would not hesitate to even add a fourth 2xx response for that case and making the description explicity explain what it is targeting  (also here your just copied "ArrayTestType  is used because it is big and long." instead of describing why the same resonse and example is there another time)

Reply:
Yes, I've added an additional 2xx response  to cover the scenario and added a more explicit description